.travis.yml

```diff
@@ -13,17 +13,28 @@ before_install:
   - sudo apt-get -qq update
   - sudo apt-get install -y rpm
   - gem install --no-ri --no-rdoc fpm
+  - go get -t -v ./...
 
 script:
   - GO111MODULE=on go test ./...
   - docker build --tag kube-bench .
   - docker run -v `pwd`:/host kube-bench install 
   - test -d cfg
   - test -f kube-bench
-
+  - go test -race -coverprofile=coverage.txt -covermode=atomic ./...
```

Comment:
I'd run `make tests` instead of duplicating tests execution command with args. So you and travis can use the same command to run tests. Pretty useful for troubleshooting failing tests locally.